### PR TITLE
[#10902] Skip 'session closed' email if already submitted

### LINKED ISSUE #10902: Skip 'session closed' emails if already submitted
Similar to #10901

### PR DESCRIPTION
**Pull request by daongochieu2810**
Fixes #10902 

Same implementation as #10906

### CODE REVIEW

File: src/main/java/teammates/logic/api/EmailGenerator.java [hunk 1]
```diff
@@ -447,9 +447,24 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructorReminders(
         List<InstructorAttributes> instructors = isEmailNeededForStudents
                                                  ? instructorsLogic.getInstructorsForCourse(session.getCourseId())
                                                  : new ArrayList<>();
-        List<StudentAttributes> students = isEmailNeededForStudents
+        List<StudentAttributes> studentsForCourse = isEmailNeededForStudents
                                            ? studentsLogic.getStudentsForCourse(session.getCourseId())
                                            : new ArrayList<>();
+        ArrayList<StudentAttributes> students = new ArrayList<>();
```
Comment (madanalogy) [inline src/main/java/teammates/logic/api/EmailGenerator.java:453]: Could similarly rename this to `studentsToEmail`

**Review by madanalogy — CHANGES_REQUESTED** [review]
There appears to be some new discussion on the issue, perhaps we could wait until it is resolved

**Review by madanalogy — APPROVED** [review]
LGTM

File: src/test/java/teammates/ui/webapi/FeedbackSessionClosedRemindersActionTest.java [hunk 2]
```diff
@@ -105,7 +105,7 @@ public void testExecute() throws Exception {
         action.execute();
 
         // 5 students and 5 instructors in course1
```
Comment (ChooJeremy) [inline src/test/java/teammates/ui/webapi/FeedbackSessionClosedRemindersActionTest.java:107]: The comment should be updated as well.

**Review by ChooJeremy — APPROVED** [review]
Github is confusing me, it says the function you edited was `generateFeedbackSessionEmailBaseForInstructorReminders` but that's not true, wonder why it's wrong.

File: src/test/java/teammates/ui/webapi/FeedbackSessionClosedRemindersActionTest.java (hunk 2 above)
Comment (daongochieu2810) [reply src/test/java/teammates/ui/webapi/FeedbackSessionClosedRemindersActionTest.java:107]: Hi Im not sure how to update the comment

Comment (ChooJeremy) [reply src/test/java/teammates/ui/webapi/FeedbackSessionClosedRemindersActionTest.java:107]: Well, right now there seems to be some conflicting information. The comment says that there are 10 people, but the next line says 8?  That means that the code and the comment is conflicting.

**Review by ChooJeremy — APPROVED** [review]

**Pull request merged**